src/FakeAuth/FakeAuth.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFrameworks>netcoreapp3.1;net6.0</TargetFrameworks>

```

Comment:
🤩🤩

Reply:
I did make an assumption that 6.0 and 3.1 were enough.